build.sbt

```diff
@@ -61,42 +62,18 @@ lazy val root = (project in file("."))
     organization := "pl.touk",
     licenses := Seq("Apache 2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
     homepage := Some(url("https://github.com/TouK/flink-scala")),
-    libraryDependencies ++= (forScalaVersion(scalaVersion.value) {
-      case (2, 12) =>
-        Seq(
-          "org.apache.flink" %% "flink-scala" % flinkV.value excludeAll(
-            ExclusionRule(organization = "org.apache.flink", name = "flink-core"),
-            ExclusionRule(organization = "org.apache.flink", name = "flink-java"),
-            ExclusionRule(organization = "org.apache.flink", name = "flink-shaded-asm-9"),
-            ExclusionRule(organization = "org.slf4j", name = "slf4j-api"),
-            ExclusionRule(organization = "com.google.code.findbugs", name = "jsr305"),
-          ),
-          "com.esotericsoftware.kryo" % "kryo" % "2.24.0" % Test,
-          "org.apache.flink" % "flink-java" % flinkV.value % Test,
-        )
-      case (2, 13) =>
-        Seq(
-          "org.apache.flink" % "flink-streaming-java" % flinkV.value % "provided",
-          "com.twitter" %% "chill" % "0.9.5" exclude("com.esotericsoftware", "kryo-shaded"),
-          "com.esotericsoftware.kryo" % "kryo" % "2.24.0" % "provided",
-        )
-    } ++ Seq(
+    libraryDependencies ++= Seq(
+      "org.apache.flink" % "flink-streaming-java" % flinkV.value % "provided",
+      "com.xebialabs.chill" %% "chill" % "0.11.1",
```

Comment:
We have to switch to [this](https://github.com/arkadius/chill/commit/fcc4c5b87f7c43889cf813a803af07f742581fed) approach because xebialabs fork uses kryo library from other maven group than flink